packages/datatrak-web/src/api/queries/useSurveyResponse.ts

```diff
@@ -24,64 +20,3 @@ export const useSurveyResponse = (
     },
   );
 };
-
-/**
- * Utility hook to fetch survey response data and populate the form with it
- */
-export const useSurveyResponseWithForm = (surveyResponseId?: string | null) => {
```

Comment:
This hook is not actually used. There is another version of it in the survey feature folder that we use instead.